nodebuilder/core/config.go

```diff
@@ -17,7 +16,6 @@ var MetricsEnabled bool
 // Config combines all configuration fields for managing the relationship with a Core node.
 type Config struct {
 	IP       string
-	RPCPort  string
 	GRPCPort string
```

Comment:
what about calling it "Port" since we assume that it will anyway be a grpc?

Reply:
sure

Reply:
fe20093b27c1dbf8ee1c3678b8e43aaf301fa238

Reply:
does it also make sense to change the flag 'core.grpc.port' to 'core.port'?

Reply:
let's keep it grpc for now. I'll update flags in https://github.com/celestiaorg/celestia-node/pull/3922